cluster/manifests/01-coredns-local/daemonset-coredns.yaml

```diff
@@ -123,8 +144,8 @@ spec:
         - --neg-ttl=60
         # send requests to the last server first, only fallback to the previous ones if it's unreachable
         - --strict-order
-        - --server=10.5.0.11#53
-        - --server=127.0.0.1#9254
+        - --server=10.5.0.11#53 # TODO: fix this for ipv6
```

Comment:
Should this be an IPv6 address?

Reply:
I think in practice this address is never used as it's a fallback if the local CoreDNS is not available. Hence we didn't notice issues in IPv6 clusters.

We should fix this, but I would say we don't have to fix it before merging this.